app/ai/voice/agents/breeze_buddy/handlers/internal/builtin_dispatcher.py

```
@@ -0,0 +1,92 @@
+
+    logger.info(
+        f"[builtin_dispatcher] Dispatching '{function_config.name}' -> '{handler_name}'"
+    )
+
+    try:
+        result = await handler(context, args)
+        # Ensure consistent return format (result, None) for global functions
+        if isinstance(result, tuple):
+            return result
+        return result, None
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)